packages/pointer_interceptor/pointer_interceptor_ios/pubspec.yaml

```diff
@@ -0,0 +1,31 @@
+name: pointer_interceptor_ios
+description: iOS implementation of the pointer_interceptor plugin.
+repository: https://github.com/flutter/packages/tree/main/packages/pointer_interceptor/pointer_interceptor_ios
+issue_tracker: https://github.com/flutter/flutter/issues?q=is%3Aissue+is%3Aopen+label%3Apointer_interceptor
+version: 0.10.0
+
+environment:
+  sdk: '>=3.1.0 <=4.0.0'
```

Comment:
I missed the `<=` here in review; it needs to be `<` since 4 would indicate breaking changes, so we don't know if it will work with 4. This is causing `publish` to fail, so I need to revert the PR.